src/routes/bounce_rules_page.jsx

```diff
@@ -78,7 +79,10 @@ export default class BounceRulesPage extends React.Component {
 
   paginate(rules) {
     const { pageIndex, pageInterval } = this.state;
-    return rules.slice(pageIndex - 1, pageIndex - 1 + pageInterval);
+    return rules.slice(
+      (pageIndex - 1) * pageInterval,
```

Comment:
let's store these two things in variables so they can show more intent and be clearer to help us understand what these two numbers are in the range